ray-operator/DEVELOPMENT.md

```diff
@@ -97,6 +99,24 @@ kubectl logs {YOUR_OPERATOR_POD} | grep "Hello KubeRay"
 * The command `make docker-build` (Step 3) will also run `make test` (unit tests).
 * Step 6 also installs the custom resource definitions (CRDs) used by the KubeRay operator.
 
+#### Run the operator outside the cluster
+
+> Note: Running the operator outside the cluster allows you to debug the operator using your IDE. For example, you can set breakpoints in the code and inspect the state of the operator.
+
+```bash
+# Step 1: Create a Kind cluster
+kind create cluster --image=kindest/node:v1.24.0
+
+# Step 2: Install KubeRay operator without deployment
+helm install kuberay-operator --set deploymentEnable=false helm-chart/kuberay-operator
```

Comment:
Why is the `helm install` step needed at all if you're not running the operator? Is it for all the other dependencies except the image?

Reply:
Some resources are still needed. For example, CRD.

Reply:
There's a make target for just installing CRD, ClusterRole  and webhook configuraitons. For development can you just run `make manifests` and then run the binary locally like you have here?

Reply:
I asked @kevin85421. He said that `make manifests` only generates CRD but does not install them into the cluster.

Reply:
@MortalHappiness try `make install`: https://github.com/ray-project/kuberay/blob/master/ray-operator/Makefile#L108